tefla/core/eval_metrices.py

```diff
@@ -0,0 +1,448 @@
+import numpy as np
+import csv
+import pandas as pd
+import math
+from sklearn import metrics,utils
+from collections import defaultdict, Counter
+from sklearn.datasets import load_iris
+from ast import literal_eval
+from statistics import mean, mode
+from bokeh.models import BasicTicker, ColorBar, LinearColorMapper,ColumnDataSource, Whisker
+from bokeh.plotting import figure, output_file, show
+from bokeh.transform import transform,factor_cmap
+from bokeh.io import output_notebook
+from collections import defaultdict
+from sklearn.preprocessing import MultiLabelBinarizer
+import traceback
+
+
+"""helper functions
+"""
+def calc_acc(tp,tn,fp,fn):
```

Comment:
@mkulariya1 write proper documentation for each function, class and class methods in the same format as this https://github.com/openAGI/tefla/blob/master/tefla/core/layers.py#L22

Reply:
done